cocos/3d/misc/create-mesh.ts

```diff
@@ -261,8 +261,165 @@ export function createMesh (geometry: IGeometry, out?: Mesh, options?: createMes
     return out;
 }
 
+function getPadding (length: number, align: number): number {
+    if (align > 0) {
+        const remainder = length % align;
+        if (remainder !== 0) {
+            const padding = align - remainder;
+            return padding;
+        }
+    }
+
+    return 0;
+}
+
+export function createDynamicMesh (primitiveIndex: number, geometry: IDynamicGeometry, out?: Mesh, options?: ICreateDynamicMeshOptions) {
```

Comment:
If we prefer user to use MeshUtils, then we shouldn't export this new function, only need to keep backward compatibility for `createMesh`